image-types/rhel8/amazon-ec2.md

```diff
@@ -0,0 +1,79 @@
+# Amazon EC2 Image
+
+This image is meant to be used in [Amazon Elastic Compute Cloud (EC2)][ec2], a
+popular cloud computing platform. It has to follow Amazon’s
+[guidelines][guidelines] and [requirements][requirements] for shared AMIs.
+
+## File Format
+
+EC2 uses Amazon Machine Images (AMIs) internally, which only be created inside
```

Comment:
Small typo:
`which only be created`
`can` ^^^

Reply:
Thanks!